README.md

```diff
@@ -1,24 +1,56 @@
-# README
+# Blurred Limes
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## About The Project
 
-Things you may want to cover:
+Blurred Limes is an application that was built for a registered user to efficiently keep their kitchen organized. This saves time, money, and reduces waste.
 
-* Ruby version
+## Project Features
 
-* System dependencies
+- Welcome Page With Application Description
+- Ability to Sign Up and Log In
+- Ability to track personal food inventory
+- Ability to input new food items, update a food item, see all food items, and delete food items
+- A page with developer images and bios
 
-* Configuration
+## Running The Project Locally
+**From The Repo:**
```

Comment:
You want to make sure someone can clone the repo and include how they would get it running on local server. Do they run yarn, bundle etc.